src/main/resources/com/playground/playground/features-hidden-layers-view.fxml

```diff
@@ -0,0 +1,189 @@
+                   <Button fx:id="add1" mnemonicParsing="false" onAction="#onAddButtonClicked" prefHeight="25.0" prefWidth="25.0" text="+" visible="false" />
+                   <Button fx:id="remove2" mnemonicParsing="false" onAction="#onRemoveButtonClicked" prefHeight="25.0" prefWidth="25.0" text="-" translateX="25.0" visible="false" GridPane.columnIndex="1" />
+                   <Button fx:id="add2" mnemonicParsing="false" onAction="#onAddButtonClicked" prefHeight="25.0" prefWidth="25.0" text="+" visible="false" GridPane.columnIndex="1" />
+                   <Button fx:id="add3" mnemonicParsing="false" onAction="#onAddButtonClicked" prefHeight="25.0" prefWidth="25.0" text="+" visible="false" GridPane.columnIndex="2" />
+                   <Button fx:id="remove3" mnemonicParsing="false" onAction="#onRemoveButtonClicked" prefHeight="25.0" prefWidth="25.0" text="-" translateX="25.0" visible="false" GridPane.columnIndex="2" />
+                   <Button fx:id="add4" mnemonicParsing="false" onAction="#onAddButtonClicked" prefHeight="25.0" prefWidth="25.0" text="+" visible="false" GridPane.columnIndex="3" />
+                   <Button fx:id="remove4" mnemonicParsing="false" onAction="#onRemoveButtonClicked" prefHeight="25.0" prefWidth="25.0" text="-" translateX="25.0" visible="false" GridPane.columnIndex="3" />
+                   <Button fx:id="add5" mnemonicParsing="false" onAction="#onAddButtonClicked" prefHeight="25.0" prefWidth="25.0" text="+" visible="false" GridPane.columnIndex="4" />
+                   <Button fx:id="remove5" mnemonicParsing="false" onAction="#onRemoveButtonClicked" prefHeight="25.0" prefWidth="25.0" text="-" translateX="25.0" visible="false" GridPane.columnIndex="4" />
+                   <Button fx:id="add6" mnemonicParsing="false" onAction="#onAddButtonClicked" prefHeight="25.0" prefWidth="25.0" text="+" visible="false" GridPane.columnIndex="5" />
+                   <Button fx:id="remove6" mnemonicParsing="false" onAction="#onRemoveButtonClicked" prefHeight="25.0" prefWidth="25.0" text="-" translateX="25.0" visible="false" GridPane.columnIndex="5" />
+                   <Label fx:id="neurons1" text="0 Neurons" visible="false" GridPane.rowIndex="11" />
+                   <Label fx:id="neurons2" text="0 Neurons" visible="false" GridPane.columnIndex="1" GridPane.rowIndex="11" />
+                   <Label fx:id="neurons3" text="0 Neurons" visible="false" GridPane.columnIndex="2" GridPane.rowIndex="11" />
+                   <Label fx:id="neurons4" text="0 Neurons" visible="false" GridPane.columnIndex="3" GridPane.rowIndex="11" />
+                   <Label fx:id="neurons5" text="0 Neurons" visible="false" GridPane.columnIndex="4" GridPane.rowIndex="11" />
+                   <Label fx:id="neurons6" text="0 Neurons" visible="false" GridPane.columnIndex="5" GridPane.rowIndex="11" />
```

Comment:
Why not generate these columns when required, rather than manually typing it out?